ui/web-v2/apps/admin/src/modules/adminNotifications.ts

```diff
@@ -127,6 +127,11 @@ export const createNotification = createAsyncThunk<
   recipient.setSlackChannelRecipient(slackChannelRecipient);
   cmd.setRecipient(recipient);
 
+  // TODO: We need to implement this on the admin console
+  // so the user can choose the language
+  recipient.setLanguage(1); // Japanese
```

Comment:
As the comment describes, I will create another PR to implement the language select menu.